src/components/BatteryXTPage/index.tsx

```diff
@@ -1,12 +1,98 @@
+import { css } from '@emotion/react';
 import type { FC } from 'react';
 
-import { Layout } from '~/components';
+import {
+  CaseStudyBottomNav,
+  CaseStudyHeaderPane,
+  CaseStudyIntroContentPane,
+  CaseStudyResultsPane,
+  CaseStudySummaryPane,
+  Layout,
+  TechCard,
+  TechListGrid,
+} from '~/components';
+import { spacing } from '~/theme';
 
 export const BatteryXTPage: FC = () => {
-  // TODO: Add description for BatteryXTPage.
+  const CaseStudyResultsResultsProps = {
+    children: [
+      {
+        heading: 'Generate inbound leads',
```

Comment:
All of the other leading verbs end with an `s`
```suggestion
        heading: 'Generates inbound leads',
```